river/cluster/dbstream.py

```diff
@@ -77,15 +77,15 @@ class DBSTREAM(base.Clusterer):
 
     Attributes
     ----------
-    n_clusters
+    _n_clusters
         Number of clusters generated by the algorithm.
-    clusters
+    _clusters
         A set of final clusters of type `DBStreamMicroCluster`. However, these are either
         micro clusters, or macro clusters that are generated by merging all $\alpha$-connected
         micro clusters. This set is generated through the offline phase of the algorithm.
-    centers
+    _centers
         Final clusters' centers.
-    micro_clusters
+    _micro_clusters
         Micro clusters generated by the algorithm. Instead of updating directly the new instance points
         into a nearest micro cluster, through each iteration, the weight and center will be modified
         so that the clusters are closer to the new points, using the Gaussian neighborhood function.
```

Comment:
You can still use the names without the `_` prefix.

Reply:
Thanks for catching this, this was by mistake. I'll resolve this shortly, in the next commit.